views/partials/requestData.pug

```diff
@@ -0,0 +1,158 @@
+h1(id='request-data-title') Destiny Hunter API
+div(class='request-data' id='request-data-1')
+  div(class='section')
+    div(class='label') Method:
+    div(class='value') GET
+  div(class='section')
+    div(class='label') Route:
+    div(class='value') /hunters
+  div(class='section')
+    div(class='label') Description:
+    div(class='value') Returns a list of all hunters in Destiny.
+
+div(class='request-data hidden' id='request-data-2')
+  div(class='section')
+    div(class='label') Method:
+    div(class='value') GET
+  div(class='section')
+    div(class='label') Route:
+    div(class='value') /hunter/{tag}
+  div(class='section')
+    div(class='label') Description:
+    div(class='value') Returns the hunter with the tag provided in the route.
+
+div(class='request-data hidden' id='request-data-3')
+  div(class='section')
+    div(class='label') Method:
+    div(class='value') GET
+  div(class='section')
+    div(class='label') Route:
+    div(class='value') /hunter/class/{subclass}
+  div(class='section')
+    div(class='label') Description:
+    div(class='value') Returns a list of hunters in Destiny with the subclass provided in the route.
+
+div(class='request-data hidden' id='request-data-4')
+  div(class='section')
+    div(class='label') Method:
+    div(class='value') POST
+  div(class='section')
+    div(class='label') Route:
+    div(class='value') /hunters
+  div(class='section')
+    div(class='label') Header:
+    div(class='value') Content-Type: application/json
+  div(class='section')
+    div(class='label') Body:
+    div(class='value')  {"tag": "zettlechips", "subclass": "Arcstrider", "grenades": "Skipbolt", "tree": "Way of the Wind", "exoticName": "Young Ahamkaras Spine"}
+  div(class='section')
+    div(class='label') Description:
+    div(class='value') Creates a new hunter from the data provided in the body.
```

Comment:
Doesn't work with the example provided, does work if I look up an existing exotic so it's just a problem with the documentation

![image](https://user-images.githubusercontent.com/5775716/84064441-9a758800-a990-11ea-8335-7b30270f463b.png)